src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -419,20 +419,19 @@ async def find_interpreter(interpreter_constraints: PexInterpreterConstraints) -
                 "-c",
                 # N.B.: The following code snippet must be compatible with Python 2.7 and
                 # Python 3.5+.
+                #
+                # When hashing, we pick 8192 for efficiency of reads and fingerprint updates
+                # (writes) since it's a common OS buffer size and an even multiple of the
+                # hash block size.
```

Comment:
Moved out of the below code snippet. That code is directly included in the argv, so it results in a long debug message with lots of noise.